documentation/staging/content/samples/credentials/_index.md

```diff
@@ -24,7 +24,7 @@ The parameters are as follows:
 
 ```  
   -u user name, must be specified.
-  -p password, must be specified.
+  -p password, must be provided using -p argument or user will be prompted to enter a value.
```

Comment:
provided using -> provided using the

Reply:
Thanks Rosemary. Updated files with the suggested edits.